.github/workflows/prepare_release_changelog.yml

```diff
@@ -39,11 +39,11 @@ jobs:
           RELEASE_TITLE="SNAPSHOT - $SHA_SHORT"
           RELEASE_TAG_NAME="v_snapshot_$SHA_SHORT"
 
-          echo "::set-output name=sha_long::$SHA_LONG"
-          echo "::set-output name=sha_short::$SHA_SHORT"
-          echo "::set-output name=changelog::$CHANGELOG"
-          echo "::set-output name=release_title::$RELEASE_TITLE"
-          echo "::set-output name=release_tag_name::$RELEASE_TAG_NAME"
+          echo "sha_long=$SHA_LONG" >> $GITHUB_OUTPUT
+          echo "sha_short=$SHA_SHORT" >> $GITHUB_OUTPUT
+          echo "changelog=$CHANGELOG" >> $GITHUB_OUTPUT
```

Comment:
Can you offer a solution to manage multiline character strings?

Reply:
I create a pull request with the necessary corrections. My apologies.